packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -296,42 +348,76 @@ export const DataGrid: React.FC<GridProps> = (props) => {
                   : undefined
               }
               title={col.label}
+              width={col.width}
+              {...(values?.allowColResizable
+                ? {
+                    onHeaderCell: () => ({
+                      width: col.width,
+                      onResize: handleResize(
+                        colIndex,
+                      ) as ReactEventHandler<any>,
+                    }),
+                  }
+                : {})}
```

Comment:
You can clean this up move the ternary inside like `onHeaderCell: values?.allowResizableColumns ? handleResize : noop`